aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/AwsPresigningDecorator.kt

```diff
@@ -224,7 +224,10 @@ class AwsPresignedFluentBuilderMethod(
             },
             "alternate_presigning_serializer_registration" to writable {
                 if (presignableOp.hasModelTransforms()) {
-                    rust(".with_operation_plugin(AlternatePresigningSerializerRuntimePlugin)")
```

Comment:
is this related to this PR?

Reply:
Reverted in 01720a953b12fc775e3f0c37d83d6f6887bba4b8